doc/api/console.markdown

```diff
@@ -74,16 +74,16 @@ Defaults to `false`. Colors are customizable, see below.
 
 ### console.time(label)
```

Comment:
Maybe call the parameter `timerName` to be in line with [mdn](https://developer.mozilla.org/en-US/docs/Web/API/Console/time)?